src/bootstrap/setup.rs

```diff
@@ -20,7 +21,26 @@ impl Profile {
     }
 
     pub fn all() -> impl Iterator<Item = Self> {
-        [Profile::Compiler, Profile::Codegen, Profile::Library, Profile::User].iter().copied()
+        [Profile::Library, Profile::Compiler, Profile::Codegen, Profile::User].iter().copied()
```

Comment:
```suggestion
        // N.B. these are ordered by how they are displayed, not alphabetically
        [Profile::Library, Profile::Compiler, Profile::Codegen, Profile::User].iter().copied()
```
The change itself is fine, but I don't want someone else to undo it because they think it's 'messy'.